test/EndToEndTests/Tests/Client/Build.Desktop/LegacyReferences/TrippinServiceReference.cs

```diff
@@ -490,6 +490,7 @@ private abstract class GeneratedEdmModel
         <Annotation Term=""Org.OData.Capabilities.V1.FilterFunctions"">
           <Collection>
             <String>contains</String>
+			<String>matchesPattern</String>

```

Comment:
nit: looks like extra whitespace before the string in some editors (maybe a tab instead of space characters)